cli/disk/ftp.go

```diff
@@ -93,6 +108,9 @@ func (l *ftpDisk) Write(path string, data []byte) error {
 
 	slog.Debug("writing to file", slog.String("path", path), slog.String("schema", "ftp"))
 	if err := l.client.Stor(path, bytes.NewReader(data)); err != nil {
+		d, _ := l.ReadDirLock(filepath.Dir(path), false)
+		slog.Debug("failed dir listing", slog.String("data", spew.Sdump(d)))
```

Comment:
Is this github actions debug code or meant to stay? The message should be more descriptive if it stays